include/reactor-uc/macros.h

```diff
@@ -237,16 +237,61 @@ typedef struct DelayedConnection DelayedConnection;
                            sizeof(self->buffer[0]), sizeof(self->buffer) / sizeof(self->buffer[0]));                   \
   }
 
-#define ENTRY_POINT(MainReactorName)                                                                                   \
+typedef struct FederatedOutputConnection FederatedOutputConnection;
+#define DEFINE_FEDERATED_OUTPUT_CONNECTION(ConnectionName, BufferType)                                                 \
+  typedef struct {                                                                                                     \
+    FederatedOutputConnection super;                                                                                   \
+    BufferType buffer[1];                                                                                              \
```

Comment:
does this suffer from the same problem that also action suffered, where values would be overwritten?

Reply:
No, it does not have a TriggerDataQueue, it is only a temporary place to store the value in case it is overwritten with other calls to `lf_set`